popup.js

```diff
@@ -72,6 +73,7 @@ $(function() {
             var update = function() {
               // Compute the first 16 base64 characters of iterated-SHA-256(domain + '/' + key, 2 ^ difficulty).
               var key = $('#key').val();
+              domain = $('#domain').val();
```

Comment:
Can you add `.toLowerCase()` here? The existing logic always converts the domain to lowercase first. Also it would be good to strip any leading and trailing whitespace.